src/components/modal/ModalHeader.ts

```diff
@@ -20,3 +21,16 @@ export const ModalHeader = styled(Box)<{ hasBorder?: true }>`
     }
   }
 `;
+
+export const HeaderTitle = styled(Typography)`
```

Comment:
Forse lo chiamerei ModalTitle. HeaderTitle mi da l'impressione di un componente che possa essere usato come Header in tutta un'app.